<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditarContato</class>
 <widget class="QDialog" name="EditarContato">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_nome">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nome:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_telefone">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Telefone:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_email">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nome">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>421</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_telefone">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>421</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_email">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>421</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4_nome_principal">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Palatino Linotype</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Agenda</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gravar_informacoes">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Gravar Informações</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_voltar">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>60</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
